espnet2/fileio/sound_scp.py

```diff
@@ -48,6 +49,24 @@ def __getitem__(self, key):
                 wav, dtype=self.dtype, always_2d=self.always_2d
             )
 
+        if pitch_aug_factor != 0:
```

Comment:
This is a bit ugly, need to refactor to an augmentation layer instead of directly conduct in sound_scp fileio